src/main/java/net/glowstone/net/GlowBufUtils.java

```diff
@@ -144,11 +142,15 @@ public static void writeMetadata(ByteBuf buf, List<Entry> entries) throws IOExce
                     buf.writeFloat((float) Math.toDegrees(angle.getZ()));
                     break;
                 case POSITION:
+                    {
```

Comment:
These two variable names clash together because the case block does not isolate the variable.
```java
case POSITION:
    BlockVector vector = (BlockVector) value; // Clashes
    buf.writeLong(Position.getPosition(vector));
    break;
case OPTPOSITION:
    if (value != null) {
        BlockVector vector = (BlockVector) value; // Clashes
        buf.writeLong(Position.getPosition(vector));
    }
    break;
```

An alternative solution would be to give a different name... I'm not sure what's the best solution.